### PR TITLE
fix node tests in subdirs

### PR DESCRIPTION
**Pull request by jakemac53**
This was using the basename of the file before, oops!

### CODE REVIEW

File: lib/src/runner/node/platform.dart [hunk 1]
```diff
@@ -194,9 +194,7 @@ class NodePlatform extends PlatformPlugin
       int socketPort,
       String precompiledPath) async {
     StackTraceMapper mapper;
-    var jsPath =
-        p.join(precompiledPath, p.basename(testPath) + ".node_test.dart.js");
-
+    var jsPath = p.join(precompiledPath, testPath + ".node_test.dart.js");
```
Comment (natebosch) [inline lib/src/runner/node/platform.dart]: single quotes for consistency with the line 2 below.

Would also prefer `'$testPath.node_test.dart.js'`

File: lib/src/runner/node/platform.dart [hunk 2]
```diff
@@ -248,7 +246,7 @@ class NodePlatform extends PlatformPlugin
     try {
       return await Process.start(settings.executable,
           settings.arguments.toList()..add(jsPath)..add(socketPort.toString()),
-          environment: {'NODE_PATH': nodePath});
+          environment: {'NODE_PATH': nodePath}w);
```
Comment (natebosch) [inline lib/src/runner/node/platform.dart]: ?

Comment (pulyaevskiy) [inline lib/src/runner/node/platform.dart]: Curious what `w` means in this case?

Comment (jakemac53) [reply lib/src/runner/node/platform.dart]: lol

Comment (jakemac53) [reply lib/src/runner/node/platform.dart]: it means my finger slipped ;)

File: lib/src/runner/node/platform.dart (hunk 1 above)
Comment (jakemac53) [reply lib/src/runner/node/platform.dart]: done, in general theres a crazy mix of single/double quotes here so eventually we might want to fix that repo-wide, but at least its consistent within this method now...

**Pull request merged**